docs/content/preview/yugabyte-cloud/cloud-secure-clusters/add-connections.md

```diff
@@ -38,29 +38,34 @@ The IP allow lists assigned to a cluster are listed under **Network Access** on
 
 ## Limitations
```

Comment:
Maybe rename it to *Quotas* or *Limits* to underline that this is intended upper bound?
Also, instruction to contact support to add more rules better fits for *Quotas* I think.